src/test/java/com/start/code/TestBubbleSort.java

```diff
@@ -0,0 +1,32 @@
+package com.start.code;
+
+import com.start.code.BubbleSort;
+
+import junit.framework.TestCase;
+
+public class TestBubbleSort extends TestCase {
+	
+	public void test0BubbleSort() { 
+        int arr[] ={2,1};
+        BubbleSort.SortMyArray(arr);//Sorting
+        for(int i = 0; i < arr.length-1; i++){
+        	assertTrue(arr[i] <= arr[i+1]);
+        }
+    } 
+	
+	public void test1BubbleSort() { 
```

Comment:
Use camelCase for method names. And avoid number in them. This method could be called, testAlreadySortedList()